pxr/usd/usdSkel/doxygen/schemas.dox

```diff
@@ -76,10 +76,10 @@ name and order joints in vectorized data. For example:
 
 \code
 def SkelAnimation "Anim" {
-    uniform token[] = ["A", "A/B"]
+    uniform token[] = ["A/B", "A"]
 }
 def Skeleton "Skel" {
-    uniform token[] = ["A/B", "A"]
+    uniform token[] = ["A", "A/B"]
```

Comment:
Updated to correctly list parent joints before child joints in the Skeleton. Switched the order in SkelAnimation too just to still show that the joint order can be different between the two prims.